metakb/transform/civic.py

```diff
@@ -0,0 +1,499 @@
+        return origin
+
+    def _get_predicate(self, proposition_type, clinical_significance):
+        """Return predicate for an evidence item.
+
+        :param str proposition_type: The proposition type
+        :param str clinical_significance: The evidence item's clinical
+            significance
+        :return: A string representation for predicate
+        """
+        predicate = None
+        # TODO: Support for other clinical_significance ?
+        #  'Resistance', 'Sensitivity/Response', 'N/A', 'Poor Outcome',
+        #  'Positive', 'Better Outcome', 'Adverse Response',
+        #  'Uncertain Significance', 'Likely Pathogenic', 'Negative',
+        #  'Loss of Function', 'Gain of Function', 'Neomorphic',
+        #  'Pathogenic', 'Dominant Negative', 'Unaltered Function',
+        #  None, 'Reduced Sensitivity', 'Unknown'
+        if proposition_type == schemas.PropositionType.PREDICTIVE.value:
+            if clinical_significance == 'Sensitivity/Response':
+                predicate = schemas.PredictivePredicate.SENSITIVITY.value
+            elif clinical_significance == 'Resistance':
+                predicate = schemas.PredictivePredicate.RESISTANCE.value
+            elif clinical_significance == 'Reduced Sensitivity':
+                predicate = \
+                    schemas.PredictivePredicate.REDUCED_SENSITIVITY.value
+            elif clinical_significance == 'Adverse Response':
+                predicate = schemas.PredictivePredicate.ADVERSE_RESPONSE.value
+        # TODO
+        elif proposition_type == schemas.PropositionType.DIAGNOSTIC.value:
+            pass
+        elif proposition_type == schemas.PropositionType.PROGNOSTIC.value:
+            pass
+        elif proposition_type == schemas.PropositionType.PREDISPOSING.value:
+            pass
+        elif proposition_type == schemas.PropositionType.FUNCTIONAL.value:
+            pass
+        elif proposition_type == schemas.PropositionType.ONCOGENIC.value:
+            pass
+        return predicate
+
+    def _get_variation_descriptors(self, variant, gene):
+        """Return a list of Variation Descriptors.
+
+        :param dict variant: A CIViC variant record
+        :param dict gene: A CIViC gene record
+        :return: A list of Variation Descriptors
+        """
+        structural_type = None
+        molecule_context = None
+        if len(variant['variant_types']) == 1:
+            # TODO: Go through SO to find the molecule_context
+            #  Is there a better way to do this?
+            so_id = variant['variant_types'][0]['so_id']
+            if so_id in ['SO:0001583', 'SO:0001818']:
+                structural_type = 'SO:0001060'
+                molecule_context = 'protein'
+            elif so_id in ['SO:0001886', 'SO:0001576', 'SO:0001889']:
+                structural_type = 'SO:0001060'
+                molecule_context = 'transcript'
+            else:
+                # TODO: Genomic
+                pass
```

Comment:
Is there a better way to find the `molecule_context`? I was thinking of going through SO and try to get a list of possible values, but SO has so many concepts and if it doesn't say `protein`, `transcript`, or `genomic` then I don't know what it classifies as. 
For CIViC, it appears that the `structural_type` is always `SO:0001060` (I'll set this in line 265 and remove lines 272 and 275).